ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_adapter_config.schema.yml

```
@@ -145,6 +145,11 @@ $defs:
         type: bool
         description: |
           Configures sFlow on the interface. Overrides `fabric_sflow` setting.
+      flow_tracking:
+        type: dict
+        $ref: "eos_designs#/$defs/flow_tracking_link"
+        description: |
+          Configures flow-tracking on the interface. Overrides `fabric_flow_tracking` setting.
```

Let's be more specific for each of the toggles. (please update the same for the others - and if you really feel gracious, you can update for sflow as well :))
```suggestion
          Configures flow-tracking on the interface. Overrides `fabric_flow_tracking.endpoints` setting.
```